src/developer_config.yaml

```diff
@@ -9,7 +9,8 @@ global:
   output_auto_outliers: False
   output_outlier_qa : False
   output_estimation_qa: False
-  output_short_form: True
+  output_short_form: False
+  dev_test : False
```

Comment:
Is this to distinguish between devtest and DAP?

Reply:
So is that to say

`hdfs == True` => we're running in either DevTest or DAP
`hdfs == True and dev_test == True` => we're only running in DevTest

Reply:
yes, that's right. Unless we decide we're not going to have the option of running in DevTest, then I can remove this "fix" for the anonymized data altogether (meaning that if in future someone does want to work in DevTest, they'd need to debug the problem of no "instance" column in the anon data again)